src/sphinx_book_theme/assets/scripts/index.js

```diff
@@ -56,7 +56,7 @@ var toggleFullScreen = () => {
  * the screen.
  */
 var scrollToActive = () => {
-  var navbar = document.getElementById("site-navigation");
+  var navbar = document.getElementsByClassName("bd-sidebar-primary")[0];
```

Comment:
I think that we should be able to delete the whole scrolltoactive section, since that code exists in the pydata theme:

https://github.com/pydata/pydata-sphinx-theme/blob/cf8a947c695211600d928bbb39bcd74b6f11ce8a/src/pydata_sphinx_theme/assets/scripts/index.js#L155

Reply:
how is it supposed to work again? the primary sidebar should scroll automatically if the active page is off-screen right?

Reply:
@choldgraf I checked using master, and this doesn't seem to be working https://github.com/executablebooks/sphinx-book-theme/pull/609 . Did it work for you?